File: sample-rxjava/src/test/java/org/dbtools/sample/model/database/TestMainDatabaseConfig.java
package org.dbtools.sample.model.database;

import org.dbtools.android.domain.AndroidDatabase;
import org.dbtools.android.domain.config.BuildEnv;
import org.dbtools.android.domain.config.TestDatabaseConfig;

import java.util.Arrays;

import javax.annotation.Nonnull;

public class TestMainDatabaseConfig extends TestDatabaseConfig {
    public TestMainDatabaseConfig(@Nonnull String databaseName) {
        super(databaseName,
                Arrays.asList(new AndroidDatabase(DatabaseManagerConst.MAIN_DATABASE_NAME, BuildEnv.GRADLE.getTestDbDir() + databaseName, DatabaseManager.MAIN_TABLES_VERSION, DatabaseManager.MAIN_VIEWS_VERSION)),
                BuildEnv.GRADLE
        );
    }
}

File: sample-basic-kotlin/src/test/kotlin/org/dbtools/sample/kotlin/TestMainDatabaseConfig.kt
package org.dbtools.sample.kotlin

import org.dbtools.android.domain.AndroidDatabase
import org.dbtools.android.domain.config.TestDatabaseConfig
import org.dbtools.sample.kotlin.model.database.DatabaseManager
import org.dbtools.sample.kotlin.model.database.DatabaseManagerConst
import java.util.*

class TestMainDatabaseConfig(databaseName: String) : TestDatabaseConfig(
        databaseName,
        Arrays.asList(AndroidDatabase(DatabaseManagerConst.MAIN_DATABASE_NAME, TEST_DB_DIR + databaseName, DatabaseManager.mainTablesVersion, DatabaseManager.mainViewsVersion)))
